shell/platform/android/io/flutter/embedding/engine/systemchannels/SpellCheckChannel.java

```diff
@@ -0,0 +1,81 @@
+package io.flutter.embedding.engine.systemchannels;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import io.flutter.Log;
+import io.flutter.embedding.engine.dart.DartExecutor;
+import io.flutter.plugin.common.JSONMethodCodec;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import java.util.ArrayList;
+import org.json.JSONArray;
+import org.json.JSONException;
+
+public class SpellCheckChannel {
+  private static final String TAG = "SpellCheckChannel";
+
+  @NonNull public final MethodChannel channel;
+  @Nullable private SpellCheckMethodHandler spellCheckMethodHandler;
+
+  @NonNull @VisibleForTesting
```

Comment:
I didn't see usages of `parsingMethodHandler` in the tests, is this annotation correct?